### ISSUE TITLE
[FEATURE] Add support for the MARS GAMING MSD-ONE

### ISSUE THREAD

**Issue by jlucasp25** [StreamController/StreamController#252]
Would be nice to have some compatibility to off-brand controllers such as the MARS GAMING MSD-ONE.

Is that a planned future feature?

**Comment by G4PLS** [comment]
No this is not planned, were using a library that basically handles the communication with the stream decks.
If I remember correctly this is the one were using: https://github.com/abcminiuser/python-elgato-streamdeck

**Comment by Core447** [comment]
If someone writes the code to add this device to the mentioned library, we can ofc also add it to StreamController. For this it doesn't matter if it's merged into `python-elgato-streamdeck` or not, as we can patch it into it. An example is [this](https://github.com/abcminiuser/python-elgato-streamdeck/pull/148) PR for the Mirabox Stream Dock 293S.

**Comment by teras** [comment]
Is the [python-elgato-streamdeck](https://github.com/abcminiuser/python-elgato-streamdeck) project in maintenance mode only? There's no activity on this PR for quite some time now.

**Comment by Core447** [comment]
> Is the [python-elgato-streamdeck](https://github.com/abcminiuser/python-elgato-streamdeck) project in maintenance mode only? There's no activity on this PR for quite some time now.

No, but I think the main dev has a lot of other thing going on.

**Comment by rescbr** [comment]
Author of that PR here and not the project's maintainer. I have to find some spare time to work on that ;)

**Comment by fightforlife** [comment]
If I wanted to test the Mirabox from this PR https://github.com/abcminiuser/python-elgato-streamdeck/pull/148 in Streamcontroller I would have todo the following, right?
1. Pull the PR in as a requirement
2. change all the PID/VID referneces here to include the Mirabox.
3. Something more?

Just tried it, got the following error:
TypeError: Can't instantiate abstract class Mirabox293S without an implementation for abstract methods 'set_key_color', 'set_screen_image'

Will try to look into it.

**Comment by fightforlife** [comment]
Added the two functions, and it is working. Nice!
Big Thanks to @rescbr. You only need to add: 

```

    def set_key_color(self, key, r, g, b):
        pass

    def set_screen_image(self, image):
        pass
```
at the end of the Mirabox file.


Edit: Created a PR DRaft for the MiraBox Device which works the same but just has a different PID/VID.
https://github.com/StreamController/StreamController/pull/318

**Comment by ImDevinC** [comment]
Based on comments, this should be working now. If not, please commend and we can reopen
